megaman/embedding/isomap.py

```diff
@@ -149,10 +147,10 @@ class Isomap(BaseEmbedding):
     random_state : numpy.RandomState or int, optional
         The generator or seed used to determine the starting vector for arpack
         iterations.  Defaults to numpy.random.RandomState
-    eigen_tol : float, optional. Tolerance for 'arpack' solver.
     path_method : string, optionl. method for computing graph shortest path.
         One of ['auto', 'D', 'FW', 'BF', 'J'].
         See `scipy.sparse.csgraph.shortest_path` for more information.
+    solver_kwds : any additional keyword arguments to pass to the selected eigen_solver
```

Comment:
Could be good to list solvers and examples of keywords they accept (if it's too long, maybe put it under ``Notes`` below and reference that here)